apps/namadillo/src/App/Staking/UnbondingAmountsTable.tsx

```diff
@@ -20,15 +21,14 @@ export const UnbondingAmountsTable = (): JSX.Element => {
   ];
 
   const rows = useMemo(() => {
-    if (!myUnbonds.isSuccess) return [];
+    if (!myValidators.isSuccess) return [];
 
     const rowsList: TableRow[] = [];
-    for (const myValidator of myUnbonds.data) {
-      const { validator, unbondedAmount, withdrawableAmount } = myValidator;
+    for (const myValidator of myValidators.data) {
+      const { validator } = myValidator;
+      const unbonding = myValidator.withdrawable.concat(myValidator.unbonding);
 
-      const amount = new BigNumber(unbondedAmount || withdrawableAmount || 0);
-
-      if (amount.gt(0)) {
+      unbonding.forEach((entry: UnbondingValidator) => {
```

Comment:
Still related with the previous comment, maybe we could rename `UnbondingValidator` to `UnbondingItem`, `UnbondingEntry`, `UnbondingRequest` or something similar

As far as I understood, the same validator XYZ could have many items unbonding at different times. Some on withdraw state. Some waiting more time than other, etc